src/events.rs

```diff
@@ -109,6 +109,25 @@ where E: Send + 'static
             }
         }
     }
+
+    /// A non-blocking receive which returns instantly.
```

Comment:
Maybe I would write,
```suggestion
    /// Attempts to receive an event without blocking
```
but only because the comment follows the same pattern as other try_recv calls as `mpsc` or `crossbeam` and the people can associate the same behavior.